app/controllers/claims/support/schools_controller.rb

```diff
@@ -1,2 +1,9 @@
 class Claims::Support::SchoolsController < Claims::Support::ApplicationController
+  def index
+    @schools = Claims::School.includes(:gias_school).order("gias_schools.name")
```

Comment:
```suggestion
    @schools = Claims::School.includes(:gias_school).order("gias_schools.name ASC")
```

Let's be explicit with the order direction.